tests/suites/events.js

```diff
@@ -301,7 +301,7 @@ test('setDate: triggers change and changeDate events', function(){
 
     this.dp.setDate(new Date(2011, 2, 5));
 
-    equal(triggered_change, 2);
+    equal(triggered_change, 0);
```

Comment:
The reason for taking it out completely is that the change event will be fired on the base element anyway won't it? Therefore if someone is targeting the base element with an `on.('change', ...)` handler, it will always be fired more than once. 

On the other hand, if someone want to specifically listen for a date change, they can listen for the `changeDate` event.

Would it be worth moving this PR to target a 2.0 branch if using semver or otherwise 1.4 to account for the BC break?